base/bitarray.jl

```diff
@@ -25,8 +25,18 @@ type BitArray{N} <: DenseArray{Bool, N}
     end
 end
 
-BitArray{N}(dims::NTuple{N,Int}) = BitArray{N}(dims...)
+# note: the docs for the two signatures are unified, but only
+# the first one is recognized by the help system; it would be nice
+# to fix this.
```

Comment:
Open an issue for this, rather than just a comment in here? (Otherwise this will probably get forgotten.)